sinfonier-backend-api/tests/data/topologies.json

```diff
@@ -0,0 +1,176 @@
+[
+  {
+    "_id": "5805f292b5aa21915447aaf9",
+    "description": "",
+    "name": "TopologyOne",
+    "status": "active",
+    "created_at": "2016-10-18T09:59:46.191+0000",
+    "updated_at": "2016-10-18T09:59:46.191+0000",
+    "sharing": false,
+    "authorId": "test@test.com",
+    "config": {
+      "wires": [
+        {
+          "xtype": "WireIt.BezierWire",
+          "src": {
+            "moduleId": 0,
+            "terminal": "out"
+          },
+          "tgt": {
+            "moduleId": 1,
+            "terminal": "in[]"
+          }
+        }
+      ],
+      "modules": [
+        {
+          "name": "Dummy",
+          "module_id": "57ff8856b5aad378bac5abda",
+          "module_version_id": "57ff8856b5aad378bac5abd9",
+          "type": "spout",
+          "language": "java",
+          "parallelisms": 1,
+          "versionCode": 1,
+          "config": {
+            "xtype": "WireIt.FormContainer",
+            "position": [
+              55,
+              28
+            ]
+          },
+          "value": {
+            "keylist": [
+            ],
+            "valuelist": [
+            ],
+            "frequency": 1,
+            "delay": 1
+          }
+        },
+        {
+          "name": "LogIt",
+          "module_id": "57ffa120b5aa858d1177618b",
+          "module_version_id": "57ffa120b5aa858d1177618a",
+          "type": "drain",
+          "language": "java",
+          "parallelisms": 1,
+          "versionCode": 1,
+          "config": {
+            "xtype": "WireIt.FormContainer",
+            "position": [
+              85,
+              217
+            ]
+          },
+          "value": {
+          }
+        }
+      ],
+      "properties": {
```

Comment:
With last changes on "develop/1.1.0", this attribute must be called "stormProperties" or "topologyProperties"